lib/formulator.ex

```diff
@@ -2,31 +2,80 @@ defmodule Formulator do
   use Phoenix.HTML
   alias Formulator.HtmlError
 
-  def input(form, field, options \\ []) do
+  @doc  """
+  Returns an html input with an associated label.
+
+  Passing `label: false` into the options will return the input without a label.
+  In that case it will add an `aria-label` attribute to the input to improve
+  accessibility.
+
+  ## Examples
+
+  Basic input:
+      <%= input form, :email %>
+      #=> <label for="user_email">Email</label>
+      #=> <input id="user_name" name="user[email]" type="text" value="">
+
+  Without a label:
+      <%= input form, :email, label: false %>
+      #=> <input id="user_name" name="user[email]" aria-label="email" type="text" value="">
```

Comment:
Agreed. I'd like to make that in another pass. My intention here was just to document the bare minimum to make sure the new `label: false` feature was documented.